src/D2L.Bmx/Program.cs

```diff
@@ -55,6 +55,9 @@
 var profileOption = new Option<string>(
 	name: "--profile",
 	description: "aws profile name" );
+var MFAOption = new Option<string>(
+	name: "--dftMFA",
+	description: "the defaultMFA in the provider_method format (eg. OKTA_question)" );
```

Comment:
Looking at the implementation in this PR and [Okta's doc on MFA API](https://developer.okta.com/docs/reference/api/factors/), I have doubt on how worthwhile this "default MFA option" feature really is.
The description here is cryptic, and hard to make it understandable too. Even if users understand this, it's still hard for users to know what to enter here (or in the config). Who would've thought of entering `GOOGLE_token:software:totp` for example?

Selecting the only MFA option when appropriate is a nice-to-have and an easy win, but I'm not convinced we should implement default MFA selection in the config or on the command line.

Reply:
Agreed, the way Okta exposes factors, names how the names are written is really no good.
Since the MFA case is already an edge case for Okta authn the value is also lessened.